Python/samples/quanser-qube/selector.ink

```diff
@@ -0,0 +1,170 @@
+###
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Multi-concept with programmed selector architecture using Quanser Qube.
+# Train the SwingUp Concept
+# Train the Balance Concept
+# Build a Programmed Concept for selecting between the two
+
+###
+
+inkling "2.0"
+using Number
+using Math
+using Goal
+
+# Constant threshold for desired region of pendulum, where 0 is vertical
+const alpha_balance_threshold = 12 
+
+# Constant threshold for defining terminal condition of motor
+const theta_rotation_threshold = 90
+
+# State from the sim and for the brain
+type ObservableState {
+    theta: number, # radians, motor angle
+    alpha: number, # radians, pendulum angle
+    theta_dot: number, # radians / s, motor angular velocity
+    alpha_dot: number, # radians / s, pendulum angular velocity
+}
+
+# Action type definition is the same for both concepts
+type BrainAction {
+    Vm: Number.Float32<-3 .. 3> # Voltage of motor
```

Comment:
Important to `Number.Float32` here or can we just do `number`? Have had issues in the past with `Number.Float32`s not being downcasted correctly to FP16 when exporting brains, so I've tried to use `number`s instead for future compatibility.

Reply:
yeah no problem!